src/genericComponents/Checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { InputGroup } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const Checkbox = (props: any) => {
+  const [clicked, setClicked] = useState(false);
+
+  return <CAGCheckbox onClick={() => setClicked(!clicked)}></CAGCheckbox>;
+};
+
+const CAGCheckbox = styled(InputGroup.Checkbox)`
+  background: var(--unnamed-color-ffffff) 0% 0% no-repeat padding-box;
```

Comment:
`var(--unnamed-color-ffffff)` doesn't work here. Just do `background: white;` for now.